main.nf

```diff
@@ -76,22 +76,22 @@ ch_synapse_files
 // Update Synapse URIs in input file with staged locations
 process update_input {
 
-  publishDir "${outdir}/${run_name}/",      mode: 'copy'
+  publishDir "${input_file.scheme}://${input_file.parent}/synstage/",  mode: 'copy'
+  publishDir "${outdir}/${run_name}/",          mode: 'copy'
```

Comment:
Does this mean the sample sheet is published in both directories?

Reply:
Yes! Documented in the tip box [here](https://nextflow.io/docs/latest/process.html#publishdir)